decidim-core/app/scrubbers/decidim/user_input_scrubber.rb

```diff
@@ -25,7 +25,7 @@ def custom_allowed_attributes
     end
 
     def custom_allowed_tags
-      Loofah::HTML5::SafeList::ALLOWED_ELEMENTS_WITH_LIBXML2 + %w(iframe)
+      Loofah::HTML5::SafeList::ALLOWED_ELEMENTS_WITH_LIBXML2 + %w(iframe style)
```

Comment:
This sanitizer is used by all other user inputs, I'm worried this would allow evil users to perform stuff. 

What do you think about using a specific input scrubber for the newsletters that allows this without modifying the `UserInputScrubber`?